test/functional/provider/python_spec.lua

```diff
@@ -15,49 +25,58 @@ do
   end
 end
 
-describe('python commands and functions', function()
-  before_each(function()
-    clear()
-    command('python import vim')
-  end)
+before_each(function()
```

Comment:
@justinmk Busted does not have thorough documentation. I know that this works locally and I know that this code works on travis and docs do not say that `before_each` *must not* be used like this (as well as that `before_each` *must* be placed inside `describe`), so I have this code. There will be more: a071e3896ecc7e2ff36fbcdbdc39e540bbca8511 (given issues with tests on branch this commit belongs to I guess I will move it to a separate PR).